samples/data-grid/basic/cell-events/demo.js

```diff
@@ -0,0 +1,28 @@
+DataGrid.dataGrid('container', {
+    table: {
+        columns: {
+            date: [1640995200000, 1641081600000, 1641168000000, 1641254400000],
+            product: ['Apples', 'Pears', 'Plums', 'Bananas'],
+            weight: [100, 40, 0.5, 200],
+            price: [1.5, 2.53, 5, 4.5],
+            metaData: ['a', 'b', 'c', 'd'],
+            icon: ['Apples URL', 'Pears URL', 'Plums URL', 'Bananas URL']
+        }
+    },
+    caption: {
+        text: 'Open the console to see the events being logged.'
+    },
+    events: {
+        cell: {
```

Comment:
Shouldn't we have a similar for the `head`?

Reply:
Maybe, but I would treat it as an idea for the future.